test/Microsoft.TemplateEngine.TestHelper/TestHost.cs

```diff
@@ -51,6 +52,7 @@ public class TestHost : ITemplateEngineHost
                 Microsoft.Extensions.Logging.LoggerFactory.Create(builder =>
                     builder
                         .SetMinimumLevel(LogLevel.Trace)
+                        .AddProvider(new InMemoryLoggerProvider(_messagesCollection))
```

Comment:
Just a food for thoughts - no change needed: since we are touching logging providers for tests, it might be good idea to add logging into XUnit output (https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/Microsoft.Extensions.Logging/tests/DI.Common/Common/src/XunitLoggerProvider.cs#L11)

Reply:
I added XUnit logger, however it needs `xunit.runner.json` added to test project, that enables diagnostic messages.